app/src/main/java/in/testpress/testpress/models/Exam.java

```diff
@@ -221,12 +221,16 @@ public void setStartDate(String startDate) {
 
     public String formatDate(String inputString) {
         Date date = null;
-        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
+        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
         simpleDateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
         try {
-            date = simpleDateFormat.parse(inputString);
-            DateFormat dateformat = DateFormat.getDateInstance();
-            return dateformat.format(date);
+            if(inputString != null && !inputString.isEmpty()) {
```

Comment:
This patch is not related with the issue. Please put it in a separate issue and commit.